src/engine/controls/cuecontrol.cpp

```diff
@@ -2656,6 +2656,10 @@ void HotcueControl::slotHotcueColorChangeRequest(double color) {
     }
     // qDebug() << "HotcueControl::slotHotcueColorChangeRequest" << color;
     m_hotcueColor->setAndConfirm(color);
+    // Note: for some reason the valueChanged() signal is not emitted,
```

Comment:
This is blocked to avoid endless loops. I think due to the read-only nature, slotHotcueColorChanged() is never called and can be inlined. setAndConfirm() should then only be set at the end when all other checks also have been performed successful.

Reply:
Please elaborate, I'm not very familiar with the contrl internals.
I see 
https://github.com/mixxxdj/mixxx/blob/94ffc14168c6e044fb36e7d32dbfaaef0026332c/src/control/control.cpp#L289-L304
This emits `valueChanged`, not a value change request. Where is that blocked? I don't see how that would create an infinite loop.

`slotHotcueColorChanged()` is indeed never called, and we also have an unused signal `hotcueColorChanged`.
I'll adjust that.

Reply:
It is blocked here: 
https://github.com/mixxxdj/mixxx/blob/94ffc14168c6e044fb36e7d32dbfaaef0026332c/src/control/controlobject.cpp#L50

Reply:
Thanks!